v2/cli/term/page_router.go

```diff
@@ -86,9 +100,14 @@ func (r *pageRouter) loadPage(pageName string, fn func()) {
 	// Build a new context for the auto-refresh goroutine to use
 	var ctx context.Context
 	ctx, r.cancelRefreshFn = context.WithCancel(context.Background())
-	r.SwitchToPage(pageName) // Bring the page into focus
-	fn()                     // Synchronously refresh the page once
-	go func() {              // Start auto-refreshing
+	if hideOtherPages {
+		r.SwitchToPage(pageName) // Focus page and hide background
+	} else {
+		r.ShowPage(pageName) // Focus page and keep background
+		r.app.SetFocus(r.logPage.logText)
```

Comment:
@krancour After consolidating the `showPage` function, I realized that I needed to set the focus to the `logText` page. I feel like it is a bit off to pass in a `textField` as a parameter since non-floating windows wouldn't need to, so it is hard-coded in for now. This may cause problems later down the line for pop-up windows that display something other than a `logText`. Do you think it would be okay to pass in a `textField` to `loadPage` as well?

Reply:
Is there a reason that `r.app.SetFocus(pageName)` wouldn't work here? Or am I misunderstanding you?

Reply:
`SetFocs()` takes in a `tview.Application` and not a string, unlike `ShowPage`

Reply:
I see. I'll take a look in the AM.